From 3eb56974d0a4ab35409777f19273823001abcc99 Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Tue, 18 Jan 2022 21:39:11 +0000
Subject: [PATCH 1/1] runtime_non-portable-distrorid-fix-alpine

Runtime doesn't generate a proper distro-rid for Alpine, leaving the
last version digit. This, treating the RID like RHEL, removes the
trailing digit

---
 eng/native/init-distro-rid.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/native/init-distro-rid.sh b/eng/native/init-distro-rid.sh
index f71aa8640b1..bef677c77d4 100644
--- a/eng/native/init-distro-rid.sh
+++ b/eng/native/init-distro-rid.sh
@@ -41,7 +41,7 @@ initNonPortableDistroRid()
             # We have forced __PortableBuild=0. This is because -portablebuld
             # has been passed as false.
             if (( isPortable == 0 )); then
-                if [ "${ID}" = "rhel" ]; then
+                if [ "${ID}" = "rhel" ] || [ "${ID}" = "alpine" ]; then
                     # remove the last version digit
                     VERSION_ID="${VERSION_ID%.*}"
                 fi
-- 
2.34.1

